src/Microsoft.Security.Utilities.Core/IdentifiableSecrets.cs

```diff
@@ -341,6 +364,16 @@ public static string GenerateCommonAnnotatedTestKey(byte[] randomBytes,
             providerReserved = new byte[providerReservedLength];
         }
 
+        if (allocationTime.Kind != DateTimeKind.Utc)
```

Comment:
We need a release note for this bug. Something along the lines of this, in your own language. See docs\ReleaseHistory.md for the content that needs updating.

BUG: Cache current date to prevent low-likelihood timing issue where year or month changes between two DateTime.UtcNow leads to the wrong encoded timestamp.

Reply:
Added in my own language :)